nn/conformer.py

```diff
@@ -0,0 +1,204 @@
+"""
+Conformer code.
+Ref: https://arxiv.org/abs/2005.08100
+"""
+
+from typing import Tuple, List, Union
+from . import Module, ModuleList, LayerRef, Linear, dropout, layer_norm, batch_norm, Conv, swish, glu, split_dims, \
+    merge_dims, pool
```

Comment:
Similar argument as in #55, I think some import like this is cleaner:
```suggestion
from .. import nn
```